internal/handlers/const.go

```diff
@@ -41,7 +41,9 @@ const unableToRegisterSecurityKeyMessage = "Unable to register your security key
 const unableToResetPasswordMessage = "Unable to reset your password."
 const mfaValidationFailedMessage = "Authentication failed, please retry later."
 
-const ldapPasswordComplexityCode = "0000052D"
+const ldapPasswordComplexityCode = "0000052D."
+
+var ldapPasswordComplexityErrors = []string{"LDAP Result Code 19 \"Constraint Violation\": 0000052D: Constraint violation - check_password_restrictions: the password is too short. It should be equal or longer than 3 characters!", "LDAP Result Code 19 \"Constraint Violation\": Password fails quality checking policy"}
```

Comment:
```suggestion
var ldapPasswordComplexityErrors = []string{"LDAP Result Code 19 \"Constraint Violation\": Password fails quality checking policy"}
var ldapPasswordComplexityCodes = []string{"0000052D"}
```

To be considered with the other part of my review.